add Readme (#1)
wormhole-foundation/demo-mayanswift: README.md

# Mayan Swift Demo 

> This guide explains how to integrate the Wormhole Mayan Swift Route from the Wormhole SDK into your application. These Route abstract the complexity of cross-chain token swaps, handling route discovery, fee estimation, and transaction construction.

Reviewer: ```suggestion
This guide explains how to integrate the Wormhole Mayan Swift Route from the Wormhole SDK into your application. This Route abstracts the complexity of cross-chain token swaps, handling route discovery, fee estimation, and transaction construction.
```

> [!IMPORTANT]

> > Mayan Swap only works on **Mainnet**. Testing on testnet environments will fail.

Reviewer: ```suggestion
> Mayan Swap only works on **mainnet**. Testing on testnet environments will fail.
```

Reviewer: or Testnet

Author: it only works on mainnet

## Prerequisites

Ensure you have the following installed:

- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed on your machine

> - A wallet with a private key, funded with native tokens on Mainnet for gas fees

Reviewer: ```suggestion
- A wallet with a private key, funded with native tokens on mainnet for gas fees
```
